mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -703,22 +703,25 @@ def __repr__(self):
 
 
 @PIPELINES.register_module()
-class IndoorPointSample(object):
```

Comment:
We should keep the API of `IndoorPointSample` and add deprecated warning in case downstream repos use it in their projects.